esmvaltool/config-references.yml

```
@@ -603,6 +607,7 @@ references:
   davini18: "Davini, P., Zenodo. doi:10.5281/zenodo.1237837, 2018"
   deangelis15nat: "DeAngelis, A. M. et al., Nature, doi:10.1038/nature15770, 2015."
   demora2018gmd: "de Mora et al., Geosci. Model Dev., doi:10.5194/gmd-11-4215-2018, 2018."
+  docquier2017cryo: " Docquier et al., The Cryosphere, https://doi.org/10.5194/tc-11-2829-2017, 2017."
```

```suggestion
  docquier2017cryo: "Docquier et al., The Cryosphere, https://doi.org/10.5194/tc-11-2829-2017, 2017."
```

Sorry, just spotted.